examples/hotel-bookings/src/components/donutChart.js

```diff
@@ -0,0 +1,78 @@
+// Code adapted (barely) from Mike Bostock's Donut chart notebook (https://observablehq.com/@d3/donut-chart/2)
+
+import * as d3 from "npm:d3";
+
+export function donutChart(data, centerText, width, colorScale) {
```

Comment:
```suggestion
export function DonutChart(data, {centerText, width, colorScale}) {
```

I'd recommend this API (needs to be changed in the calling code too).

Reply:
Updated! Thanks Fil